### DISCUSSION TITLE
missing css class for -webkit

### DISCUSSION THREAD

**Discussion by boniface193** [tailwindlabs/tailwindcss#11724] [Ideas]
while working with the tailwind css, i tried creating a h1 with a large font that will have a background image as a fill color and i could not get the exalt class from tailwind and i thought it should be added because i use it almost every time when i am designing a website, so as everyone who might need the feature.

**Comment by mybearworld** [comment]
Are you looking for `bg-clip-text`? https://play.tailwindcss.com/p1P53dxla6

**Comment by Brittmeas** [comment]
Hello,

In Tailwind CSS, the "exalt" class you mentioned does not exist by default. However, you can easily create your own custom class to achieve the desired effect of a large font with a background image fill. Tailwind CSS is highly customizable, allowing users to extend its utility classes with their own configurations. By adding a custom class, you can streamline your workflow and apply the "exalt" class or any other frequently used styles across different projects.  [Goldsmiths Customer Service](https://www.surveyzo.com/goldsmiths-customer-service/)

**Reply by mybearworld** [reply]
...they just misspelled exact
